docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Deploy SAP NetWeaver and SAP HANA on Azure
+description:  This Microsoft Azure reference architecture shows a set of proven practices for running SAP HANA in a high availability environment on Azure.
+author: njray
+ms.date: 6/08/17
+---
+
+# Deploy SAP NetWeaver and SAP HANA on Azure
+
+This reference architecture shows a set of proven practices for running SAP HANA in a high availability environment on Azure. 
+
+![0][0]
+Figure 1. SAP HANA architecture using Microsoft Azure.
+
+\* For details about SMLG and Azure Load Balancer, see the “Load balancer” item
+in the following “Architecture” section.
+
+> [!NOTE]
+-   SAP HANA is certified for production OLAP workloads on Azure GS5-series
+    virtual machines. This reference architecture is for Azure virtual machines
+    in the G-series and M-series and differs from [SAP HANA on Azure Large Instances][azure-large-instances].
+
+-   Deployment of this reference architecture requires appropriate licensing of
+    SAP products and other non-Microsoft technologies.
+
+-   For information about the partnership between Microsoft and SAP, see [SAP HANA on Azure][sap-hana-on-azure]. 
+>
+
+## Architecture
+
+This reference architecture depicts a distributed installation of SAP applications. There are several databases that SAP applications can use, and here the SAP HANA database is described. Although this document focuses on a highly available environment within a single Azure region, all components of this architecture can be replicated to secondary regions for disaster recovery purposes.
+
+The architecture consists of the following components.
+
+-   **Virtual network**. The Azure Virtual Network service securely connects
+    Azure resources to each other. A virtual network is a representation of your
+    own network in the cloud—that is, a logical isolation of the Azure cloud
+    dedicated to your subscription. You can also connect virtual networks to
+    your network on premises.
+
+-   **Subnets**. A subnet defines a logical collections of network resources by
+    specifying address ranges within a virtual network. Subnets make it easier
+    to manage and segregate network traffic without impacting performance. For
+    example, you can create a separate subnet for each tier: application (SAP
+    NetWeaver), database (SAP HANA), and management (the jumpbox).
+
+-   **Availability sets**. These are groups of virtual machines performing the
+    same role. Using availability sets improves the availability of a service
+    during system failure or routine cloud platform maintenance.
+
+-	**Load balancers.** The SAP Web Dispatcher component is used as a load balancer 
+    for SAP traffic among the SAP application servers. To achieve high availability,
+    the [Azure Load Balancer][azure-lb] (shown on the left in Figure 1) is used
```

Comment:
Are there two failover clusters? We should add a bullet item about the failover clusters and possibly add these to the diagram